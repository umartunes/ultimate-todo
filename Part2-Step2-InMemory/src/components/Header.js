import React from 'react';


const Header = () => (
    <React.Fragment>
        <header>
            <h4 className="white-text marginZero" style={{ position: " relative", top: "75%", textIndent: "8px" }}>The Ultimate To-Do</h4>
        </header>
    </React.Fragment>
)

export default Header;