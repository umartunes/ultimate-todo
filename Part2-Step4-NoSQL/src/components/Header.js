import React from 'react';


const Header = () => (
    <React.Fragment>
        <header>
            <h4 className="white-text marginZero" style={{ position: " relative", top: "60%", textIndent: "15px" }}>The Ultimate Todo App</h4>
           
        </header>
    </React.Fragment>
)

export default Header;