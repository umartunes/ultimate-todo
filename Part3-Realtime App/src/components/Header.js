import React from 'react';


const Header = () => (
    <React.Fragment>
        <header>
            <h4 className="white-text marginZero" style={{ position: " relative", top: "60%", textIndent: "15px" }}>Ultimate Todo App</h4>
            <h6 id="today" className="white-text" style={{ position: "relative", top: "57%", textIndent: "15px" }}></h6>
        </header>
    </React.Fragment>
)

export default Header;