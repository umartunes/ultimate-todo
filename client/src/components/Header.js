import React from 'react';


const Header = () => (
    <React.Fragment>
        <header>
            <h4 className="white-text marginZero" style={{ position: " relative", top: "60%", textIndent: "15px" }}>My Day</h4>
            <h6 id="today" className="white-text" style={{ position: "relative", top: "57%", textIndent: "15px" }}>Today</h6>
        </header>
    </React.Fragment>
)

export default Header;