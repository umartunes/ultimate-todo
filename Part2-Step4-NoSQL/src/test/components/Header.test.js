import React from 'react';
import {shallow} from 'enzyme';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

// test('Should Render Header Correctly',()=>{
//     const wrapper = shallow(<Header/>);
//     expect(wrapper).toMatchSnapshot();
// })

test('Should Render Header Correctly',()=>{
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})