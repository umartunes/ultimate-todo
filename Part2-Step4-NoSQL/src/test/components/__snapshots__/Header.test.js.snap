// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Header Correctly 1`] = `
<Unknown>
  <header>
    <h4
      className="white-text marginZero"
      style={
        Object {
          "position": " relative",
          "textIndent": "15px",
          "top": "60%",
        }
      }
    >
      The Ultimate Todo App
    </h4>
  </header>
</Unknown>
`;
